eclair-node/src/test/scala/fr/acinq/eclair/api/JsonSerializersSpec.scala

```diff
@@ -76,14 +73,12 @@ class JsonSerializersSpec extends FunSuite with Matchers {
   }
 
   test("type hints") {
-    implicit val formats = JsonSupport.formats.withTypeHintFieldName("type") + CustomTypeHints(Map(classOf[PaymentSettlingOnChain] -> "payment-settling-onchain")) + new MilliSatoshiSerializer
     val e1 = PaymentSettlingOnChain(UUID.randomUUID, 42 msat, randomBytes32)
-    assert(Serialization.writePretty(e1).contains("\"type\" : \"payment-settling-onchain\""))
+    assert(Serialization.writePretty(e1)(JsonSupport.formats).contains("\"type\" : \"payment-settling-onchain\""))
```

Comment:
```suggestion
    assert(JsonSupport.serialization.writePretty(e1)(JsonSupport.formats).contains("\"type\" : \"payment-settling-onchain\""))
```

Reply:
Actually we should replace all the instances of `org.json4s.jackson.Serialization` with `JsonSupport.serialization` otherwise our test would be overriding the serialization backend for json4s.